Tools/environment_install/install-prereqs-windows-andAPMSource.ps1

```diff
@@ -4,25 +4,22 @@ Import-Module BitsTransfer
 
 Write-Output "Starting Downloads"
 
-Write-Output "Downloading MAVProxy (1/7)"
+Write-Output "Downloading MAVProxy (1/6)"
```

Comment:
I'm *sure* powershell must have the concept a variable which could hold the number being changed on all these lines :-)